src/views/components/landing-top.html

```diff
@@ -53,6 +53,7 @@ <h2>free, fast, and reliable</h2>
 							<c-slider active="{{activeSlide}}" direction="vertical" distance="500">
 								<c-slide>
 									<div class="docs-code-slide">
+										<p class="docs-comment">// NPM CDN</p>
```

Comment:
These look ugly and just duplicate the text directly below.

Reply:
Its for SEO. Its the best and most relevant place I could think to add keywords

Reply:
Rephrased and moved to a better place.